core/primitives/src/congestion_info.rs

```diff
@@ -318,6 +320,50 @@ impl CongestionInfo {
     }
 }
 
+// TODO remove default
```

Comment:
Maybe `BlockCongestionInfo::empty()`?

Reply:
Ah this was added while I was still ambitious enough to pass the information about the `all_shards` explicitly and make it an invariant instead of implicitly relying on the keys of shards congestion info and worrying what happens at the protocol upgrade. Since then I lost all of that ambition and it would be yet another refactor which I want to avoid that in the new, frozen, world. 🥶 

I'll remove the todo, it's no longer needed.